courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<p>Same as above. I don't think we will cover estimator anymore, won't we?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Great! </p><p> </p><p>nit: I'd use a more concrete name (see comment above) like create_test_dataset (resp. create_train_dataset)</p>

Reply:
Whoops, sorry. I meant to do that in the last commit.